Integrations/QuinC/README.md

```diff
@@ -0,0 +1,276 @@
+JobID: 993
+
+### accessdata-read-casefile
+---
+Reads file from case folder and puts its contents to current context
+##### Base Command
+
+`accessdata-read-casefile`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| filepath | Path to case file | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Accessdata.File.Contents | string | Contents of the file | 
+
+
+##### Command Example
+`accessdata-read-casefile filepath="\\X.X.X.X\D$\Program Files\AccessData\QuinC\app\demo\Demo Case\c00a2abf-1076-412b-8dea-67305fb8015f\Jobs\job_987\f6fac193-89ff-4f3f-92ac-0871c30621c0\1\snapshot.xml"`
+
+##### Context Example
+```
+{
+    "Accessdata.File.Contents": "<?xml version=\"1.0\"?>\r\n<root>\r\n<Process resultitemtype=\"15\"><Name>addm.exe</Name><Path/><StartTi ... ress>0</baseAddress><ImageSize>0</ImageSize><ProcessName/><FromAgent/></DLL>\r\n</root>\r\n"
+}
+```
+
+##### Human Readable Output
+<?xml version="1.0"?>
+<root>
+<Process resultitemtype="15"><Name>addm.exe</Name><Path/><StartTi ... ress>0</baseAddress><ImageSize>0</ImageSize><ProcessName/><FromAgent/></DLL>
+</root>
+
+
+### accessdata-jobstatus-scan
+---
+Checks status of the job
+##### Base Command
+
+`accessdata-jobstatus-scan`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| caseJobID | Concatenated CaseID and JobID (like "1_800") | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| CaseID | string | Case ID | 
+| ID | string | Job ID | 
+| CaseJobID | string | Concatenated CaseID and JobID (like "1_800") | 
+| State | string | State of job's execution | 
+
+
+##### Command Example
+`accessdata-jobstatus-scan caseJobID=1_987`
+
+##### Context Example
+```
+{
+    "Accessdata.Job": {
+        "ID": "987", 
+        "CaseID": "1", 
+        "State": "Success", 
+        "CaseJobID": "1_987"
+    }
+}
+```
+
+##### Human Readable Output
+Current job state: Success
+
+### accessdata-get-jobstatus-processlist
+---
+Get snapshot path from result of the process list job
+##### Base Command
+
+`accessdata-get-jobstatus-processlist`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| caseID | ID of the case | Required | 
+| jobID | ID of the job | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| State | string | Job state | 
+| Result | string | Job result | 
+
+
+##### Command Example
+`accessdata-get-jobstatus-processlist caseID=1 jobID=987`
+
+##### Context Example
+```
+{
+    "Accessdata.Job": {
+        "ID": "987", 
+        "Result": {
+            "SnapshotDetails": {
+                "File": "\\\\X.X.X.X\\D$\\Program Files\\AccessData\\QuinC\\app\\demo\\Demo Case\\c00a2abf-1076-412b-8dea-67305fb8015f\\Jobs\\job_987\\f6fac193-89ff-4f3f-92ac-0871c30621c0\\1\\snapshot.xml"
```

Comment:
Do `Result` have the same structure all them ? Like
`Result.SnapshotDetails.File`
or `Result` structure is dynamic? 
If the structure is the same, then you should declare those outputs in `Context Outputs`
`Result.SnapshotDetails.File`